packages/e2e-tests/run-e2e.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+cd ../devtools
+yarn start:shared-ganache & 
+ganache=$!
+
+cd ../$1 && yarn run wait-on ../../.ganache-deployments/ganache-deployments-8545.json
+yarn start &
+app=$!
+
+cd ../xstate-wallet
+yarn start &
+wallet=$!
+
+cd ../simple-hub
+yarn hub:watch &
+hub=$!
+
+cd ../e2e-tests
+yarn run wait-on -d 5000 http://localhost:3000 http://localhost:3055
+yarn test $1
+code=$?
+
+kill $ganache
+kill $wallet
+kill $app
+kill $hub
+killall node
```

Comment:
After spending some time trying to track all the descendant processes carefully and clean them up, I decided to use this command to make sure. It's not ideal, but we can remove it once we ensure all of our various start scripts clean up after themselves properly when they finish or are killed.